ResourceManager/IHotkeySettingsReader.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ResourceManager
+{
+    public interface IHotkeySettingsReader
+    {
+        HotkeyCommand[] LoadHotkeys(string hotkeySettingsName);
+    }
+}
```

Comment:
I've been avoiding this change, as it should be done across the codebase. HOWEVER, this will result in merge conflicts for all open PRs. (I've lived through this change in other repos).

I'm happy to do it as a separate PR.